src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/Services/EnvironmentService.cs

```diff
@@ -1,12 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 using System.Runtime.InteropServices;
 
-namespace Microsoft.DotNet.Scaffolding.Helpers.Services.Environment;
-
+namespace Microsoft.DotNet.Scaffolding.Core.Services;
 /// <summary>
 /// Wrapper over System.Environment abstraction for unit testing.
 /// </summary>
-internal class EnvironmentService : IEnvironmentService
+public class EnvironmentService : IEnvironmentService
```

Comment:
These implementations really have to be public because dotnet-scaffold uses them (we could make them part of the default services in Core otherwise). But that means dotnet-scaffold currently has a dependency on Core (dotnet-scaffold -> Helpers -> TextTemplating -> Core) which will be broken with further refactoring (and because we didn't want it to have it. Then where would these live? Maybe shared files (they'd still have binary compat via the interface so I think it'd be ok?)? Not sure what exactly to do there.